### PR TITLE
Continuous deployment

### PR DESCRIPTION
**Pull request by Adaickalavan**
Added Github Actions workflow files to automate the

+ building and pushing of SMARTS binary wheel to PyPI, and
+ building and pushing of Docker image to Docker Hub

upon release of new code versions.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -77,7 +77,7 @@ panda3d==1.10.7
 panda3d-gltf==0.10
 panda3d-simplepbr==0.7
 pandas==1.1.3
-Pillow==8.0.0
+Pillow==7.2.0
```
Comment (Gamenot) [inline requirements.txt:80]: This should fix the problem that the ULTRA team is having with this dependency right?

**Comment by Gamenot** [comment]
I do not have access to this so we may have to ask @junluo-huawei: https://docs.github.com/en/actions/reference/encrypted-secrets.

**Review by Gamenot — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (Adaickalavan) [reply requirements.txt:80]: Yes, I have updated the requirements to:
+ Pillow==7.2.0
+ chardet==3.0.2

**Review by JingfeiPeng — APPROVED** [review]

**Comment by JingfeiPeng** [comment]
When are the two scripts run?

File: .github/workflows/cd-docker.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  docker:
+    name: Build and push Docker image to Docker Hub
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v1
```
Comment (JingfeiPeng) [inline .github/workflows/cd-docker.yml:15]: what is `docker/build-push-action`? is this a script we wrote or some built-in script

**Pull request merged**